### PR TITLE
fix copyright

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@geirst please review and merge

### CODE REVIEW

File: tests/performance/hamming_distance/min_hamming_distance.rb [hunk 1]
```diff
@@ -62,11 +62,8 @@ def smoke_test
 
   def run_benchmarks(query_file, legend)
     node = (vespa.qrserver["0"] or vespa.container.values.first)
-    tmp_query_dir = in_tmp('qd')
-    node.copy(query_file, tmp_query_dir)
-    qf = tmp_query_dir + '/' + File.basename(query_file)
-    puts "qf: #{qf}"
-    run_fbench(node, 48, 20, qf, legend)
+    puts "qf: #{query_file}"
+    run_fbench(node, 48, 20, query_file, legend)
```
Comment (geirst) [inline tests/performance/hamming_distance/min_hamming_distance.rb:66]: Was this change intentional?

**Review by geirst — APPROVED** [review]
👍

File: tests/performance/hamming_distance/min_hamming_distance.rb (hunk 1 above)
Comment (arnej27959) [reply tests/performance/hamming_distance/min_hamming_distance.rb:66]: Yes…
